### ISSUE TITLE
Some Problem With Geopotential(Z).

### ISSUE THREAD

**Issue by EnumaCaliber** [neuralgcm/neuralgcm#451]
Hi NeuralGCM team,
When I used Checkpoint to perform prediction tests, the predictions of other variables were not a significant issue, but I found that the projections of geopotential were quite different. 
I used the following code to get the ground and predicted geopotential, and then drew a histogram of the data distribution.
`era5_path = 'gs://gcp-public-data-arco-era5/ar/full_37-1h-0p25deg-chunk-1.zarr-v3'
full_era5 = xarray.open_zarr(
    era5_path, chunks=None, storage_options=dict(token='anon')
)

demo_start_time = '2020-02-14'
demo_end_time = '2020-02-16'
data_inner_steps = 12  # process every 18th hour

sliced_era5 = (
    full_era5
    [model.input_variables + model.forcing_variables]
    .pipe(
        xarray_utils.selective_temporal_shift,
        variables=model.forcing_variables,
        time_shift='12 hours',
    )
    .sel(time=slice(demo_start_time, demo_end_time, data_inner_steps))
    .compute()
)

era5_grid = spherical_harmonic.Grid(
    latitude_nodes=full_era5.sizes['latitude'],
    longitude_nodes=full_era5.sizes['longitude'],
    latitude_spacing=xarray_utils.infer_latitude_spacing(full_era5.latitude),
    longitude_offset=xarray_utils.infer_longitude_offset(full_era5.longitude),
)
regridder = horizontal_interpolation.ConservativeRegridder(
    era5_grid, model.data_coords.horizontal, skipna=True
)
eval_era5 = xarray_utils.regrid(sliced_era5, regridder)
eval_era5 = xarray_utils.fill_nan_with_nearest(eval_era5)

inner_steps = 12
outer_steps =  3 * 24 // inner_steps
timedelta = np.timedelta64(1, 'h') * inner_steps
times = (np.arange(outer_steps) * inner_steps)

# initialize model state
inputs = model.inputs_from_xarray(eval_era5.isel(time=0))
input_forcings = model.forcings_from_xarray(eval_era5.isel(time=0))
rng_key = jax.random.key(42)
initial_state = model.encode(inputs, input_forcings, rng_key)

# use persistence for forcing variables (SST and sea ice cover)
all_forcings = model.forcings_from_xarray(eval_era5.head(time=1))

# make forecast
final_state, predictions = model.unroll(
    initial_state,
    all_forcings,
    steps=outer_steps,
    timedelta=timedelta,
    start_with_input=True,
)
predictions_ds = model.data_to_xarray(predictions, times=times)

# Selecting ERA5 targets from exactly the same time slice
target_trajectory = model.inputs_from_xarray(
    eval_era5
    .thin(time=(inner_steps // data_inner_steps))
    .isel(time=slice(outer_steps))
)
target_data_ds = model.data_to_xarray(target_trajectory, times=times)

u_predict = predictions_ds.geopotential.values.flatten()
u_target = target_data_ds.geopotential.values.flatten()`

<img width="1488" height="590" alt="Image" src="https://github.com/user-attachments/assets/dfaa9205-f724-4be4-9010-d1e500250cff" />
What I want to clarify is whether geopotential is not one of the predicted variables. However, I also found in the training code that the loss includes z, which is geopotential. Since I am reproducing the train code myself, I may need to confirm whether z is also not included in one of the losses; otherwise, the loss will become very large. The following is the loss code
`            prediction, target = batch_trajectory_fn(
                # The `target` and prediction returned are TrajectoryRepresentations.
                # So don't just re-use the arg `target`.
                params,
                rng,
                target,
                forcing_data,
            )
            # dimensions (batch, ensemble)
            jax.config.update("jax_disable_jit", True)

            compute_timestep_rmse(prediction, target)

            jax.config.update("jax_disable_jit", False)
            per_example_loss = batch_ensembled_loss_fn(prediction, target)`

I'm looking forward to hearing back from you. Thank you.
